change/@fluentui-react-tabs-00427f46-b6ea-45d8-989f-a7e2d94bd0bf.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32374&commitId=ec5da997f7af4dcbbcfdfca5a3a05f4a48e5a054&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Badge Mask RTL.chromium.png|7| Changed|
|Avatar Converged.Basic High Contrast.chromium.png|2| Changed|
|Avatar Converged.badgeMask.chromium.png|15| Changed|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay Dark Mode.chromium.png|991| Changed|
|Drawer.overlay drawer full.chromium.png|1138| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>